_pytest/main.py

```diff
@@ -33,6 +33,9 @@ def pytest_addoption(parser):
     parser.addini("testpaths", "directories to search for tests when no files or directories are given in the "
                                "command line.",
                   type="args", default=[])
+    parser.addini("rootdir", "define root directory for tests. If this parameter defined command argument "
```

Comment:
In general we should avoid adding both an ini option and command-line option for the same thing, because it is easy to change the value of an ini option in the command line (using `-o`) and one can change command-line options by setting `addopts` in an ini file, so it is a bit redundant.

In this case I think we will have to go with only a command-line option, because we determine the `rootdir` very early, even before we are setup to parse the `ini` file.